enterprise/dev/ci/internal/ci/pipeline-steps.go

```diff
@@ -64,13 +64,16 @@ func addWebApp(pipeline *bk.Pipeline) {
 		bk.Cmd("dev/ci/codecov.sh -c -F typescript -F unit"))
 }
 
+// Don't download browser, we use "download-puppeteer-browser" script instead
```

Comment:
```suggestion
// We provide our own Chromium instance that is installed through the `download-puppeteer-browser` script
```

This reads a bit better now that we've changed the code to no longer reference skip download, or the script